Startup Spotlight: Puzzle (#7620)
PostHog/posthog.com: contents/spotlight/startup-puzzle.md

---
title: "How Puzzle defined a buyer persona and built around it"

> date: 2024-02-01

Author: ```suggestion
date: 2024-02-12
```

author: ["joe-martin"]
showTitle: true
rootpage: /blog
sidebar: Blog
hideAnchor: true
featuredImage: ../images/blog/startup_puzzle.jpg
featuredImageType: full
category: Startups
---

> Each month, we speak to a team enrolled in PostHog for Startups about the way they work. This time, we spoke to [Puzzle](https://puzzle.io/), an accountancy platform for early-stage startups, about how they define and use their Ideal Customer Persona (ICP). 

> Building an Ideal Customer Persona (sometimes known as an Ideal Buyer Profile) is one of the most challenging but essential steps for growing your product. Not only will it help you make product and pricing decisions, it will also clarify your value proposition and dictate how you judge successes and failures.

Reviewer: We generally refer to ICP as ideal customer profile (ICP), not a persona, and a buyer profile is something different. Later on in the piece they talk about their ICP as a person, so kind of sounds like they have a buyer persona, not an ICP? 

In any case, I think it's important we're internally consistent about what ICP means to us, so maybe we frame this around the buyer persona bit and not the ICP bit?

> We’ve previously documented [how we defined our ICP](/founders/creating-ideal-customer-profile), but not all companies approach the task in the same way. So, we spoke to the team at [Puzzle](https://puzzle.io/) about how they defined their ICP, what impact the definition had, and what recommendations they had for others...

Author: ```suggestion
We’ve previously documented [how we defined our personas](/founders/creating-ideal-customer-profile), but not all companies approach the task in the same way. So, we spoke to the team at [Puzzle](https://puzzle.io/) about how they defined their persona, what impact the definition had, and what recommendations they had for others...
```

## 1. Collect both qualitative and quantitative data

> An ICP is, fundamentally, a snapshot of your ideal customer based on a gestalt of many customers. In order to define your ICP you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.

Reviewer: Can we use a simpler word than gestalt?

Author: ```suggestion
An ICP is, fundamentally, a snapshot of your ideal customer based on a broad understanding of many customers. In order to define your ICP you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.
```

> An ICP is, fundamentally, a snapshot of your ideal customer based on a broad understanding of many customers. In order to define your ICP you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.

Author: ```suggestion
A persona is, fundamentally, a snapshot of your ideal customer based on a broad understanding of many customers. In order to define your persona you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.
```

> Once you’ve done that, says Scott, you need to pair that intuitive understanding with quantitative data so you can understand if these behaviors exist in isolation or at scale. 

Reviewer: ```suggestion
Once you’ve done that, says Scott, you need to pair that intuitive understanding with quantitative data, so you can understand if these behaviors exist in isolation or at scale. 
```

> A persona is, fundamentally, a snapshot of your ideal customer based on a broad understanding of many customers. In order to define your persona you therefore need to start by speaking to your existing users directly and building an intuitive understanding of their needs.

Reviewer 2: ```suggestion
A persona is, fundamentally, a snapshot of your ideal customer based on a broad understanding of many customers. To define your persona you must start by speaking to your existing users directly and building an intuitive understanding of their needs.
```

> “We’re constantly standing next to founders who are just starting out and I do a healthy amount of interviews,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”

Reviewer 3: “We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”

Author: ```suggestion
“We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”
```

> “We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your ICP.”

Reviewer 2: I feel a bit confused if we introduce ICP as a concept to this article, can we just use persona?
```suggestion
“We’re constantly standing next to founders at each stage of the startup journey interviewing and learning about their experience,” says Scott. “But it’s one thing for a customer to say they want, need, or plan to do X, Y, or Z — and another thing if they actually do it. That’s where quantitative data is useful. You need both to create your persona.”
```

> **Putting it into practice:** It can help to [define a small number of reference customers](/founders/creating-ideal-customer-profile#5-reference-customers). Spend time with them to gather subjective data, while using a tool such as [PostHog](/startups) to collect hard facts.

## 2. Understand how your users _feel_

Armed with this qualitative and quantitative data, you can start to establish your value proposition.

> For Puzzle, this was critical for differentiating them from other products in the market. Every company needs an accounting platform, but not all founders feel the same about this need — and by focusing on users’ unaddressed needs Puzzle can identify a strong, unique position for the product.

Reviewer 2: ```suggestion
For Puzzle, this was critical for differentiating them from other products in the market. Every company needs an accounting platform, but not all founders feel the same about this need. By focusing on users’ unaddressed needs, Puzzle can identify a strong, unique position for the product.
```

> “Founders don’t sign up to Puzzle because they just can’t wait to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant and to be ready for investor reporting... We learned that they care about efficiency, accuracy and, as they generate more revenue, delaying hires in the finance department.”
> 

Author: ```suggestion
“More often than not, founders aren’t signing up to Puzzle because they’re excited to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant, gain a deeper understanding of their company’s finances, and be ready for things like investor reporting. While the needs of founders often change or mature at each stage of their journey, what’s common is that they care about compliance, efficiency, and accuracy.”

```

> “Founders don’t sign up to Puzzle because they just can’t wait to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant and to be ready for investor reporting...We learned that they care about efficiency, accuracy and, as they generate more revenue, delaying hires in the finance department.”

Reviewer: ```suggestion
“Founders don’t sign up to Puzzle because they just can’t wait to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant and to be ready for investor reporting... We learned that they care about efficiency, accuracy and, as they generate more revenue, delaying hires in the finance department.”
```

> “Founders don’t sign up to Puzzle because they just can’t wait to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant and to be ready for investor reporting...We learned that they care about efficiency, accuracy and, as they generate more revenue, delaying hires in the finance department.”

Reviewer 3: “More often than not, founders aren’t signing up to Puzzle because they’re excited to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant, gain a deeper understanding of their company’s finances, and be ready for things like investor reporting. While the needs of founders often change or mature at each stage of their journey, what’s common is that they care about compliance, efficiency, and accuracy.”

> All of this is reflected in Puzzle’s online messaging, with language which directly addresses founders ahead of accountants and emphasizes how the product integrates simply with other tools which founders already use. 

Reviewer 2: ```suggestion
All of this is reflected in Puzzle’s messaging, with language which directly addresses founders ahead of accountants and emphasizes how the product integrates with the other tools founders already use. 
```

> > **Putting it into practice:** [We advise having an approximate structure for user interviews](/blog/how-to-work-out-what-users-need), but leave room to flex. If users go off on a tangent during an interview that could give you vital [insights to their actual feelings](/handbook/product/user-feedback)!

Reviewer 2: ```suggestion
> **Putting it into practice:** [Have a structure for user interviews](/blog/how-to-work-out-what-users-need), but leave room to flex. If users go off on a tangent during an interview that could give you vital [insights to their actual feelings](/handbook/product/user-feedback)!
```

## 3. Build pricing around user segments

Next, you can delve into the quantitative data by looking at how users can be segmented into different groups based on criteria such as size or industry. Puzzle used this information to structure its pricing model, for example. 

> “On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and a scale plan for companies which are generating revenue.

Reviewer 3: “On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and an automation plan for companies which are a little more mature and prioritize efficiency, accuracy, and not waiting weeks to close their books.

Author: ```suggestion
“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and an automation plan for companies which are a little more mature and prioritize efficiency, accuracy, and not waiting weeks to close their books."
```

> Next, you can delve into the quantitative data by looking at how users can be segmented into different groups based on criteria such as size or industry. Puzzle used this information to structure its pricing model, for example. 

Reviewer 2: ```suggestion
Next, you can delve into the quantitative data by segmenting users based on criteria such as size or industry. Puzzle used this information to structure its pricing model differently for early-stage founders versus mature startups, for example. 
```

> Each plan also reflects the pain points a user would deal with at that stage too and addresses it with a product feature. Monthly compliance checklists in the free plan, for example, help formation-stage founders get started quickly — but give way in the scale-up plan to AI automation and new processes.

Reviewer 3: Each plan also reflects the pain points a user would deal with at that stage too and addresses it with a product feature. Monthly compliance checklists in the free plan, for example, help formation-stage founders get started quickly — but give way in the Automation plan to AI and streamlined workflows.

Author: ```suggestion
Each plan also reflects the pain points a user would deal with at that stage too and addresses it with a product feature. Monthly compliance checklists in the free plan, for example, help formation-stage founders get started quickly — but give way in the Automation plan to AI and streamlined workflows.
```

> > **Putting it into practice:** If you aren’t using delineated pricing plans then it can still be helpful to think of your broader [pricing principles](/handbook/engineering/feature-pricing). Ask yourself: how do these principles work for your different user segments? 

Reviewer: Delineated? Is there a simpler way to say this?

Author: ```suggestion
> **Putting it into practice:** If you aren’t using tiered pricing plans then it can still be helpful to think of your broader [pricing principles](/handbook/engineering/feature-pricing). Ask yourself: how do these principles work for your different user segments? 
```

> ## 4. Reinforce your ICP with internal teams

Author: ```suggestion
## 4. Reinforce your persona to internal teams
```

> As you grow it’s important not to drift from your ICP or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 

Reviewer: ```suggestion
As you grow, it’s important not to drift from your ICP or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 
```

> As you grow, it’s important not to drift from your ICP or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 

Author: ```suggestion
As you grow, it’s important not to drift from targeting your persona or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 
```

> “When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “Then I saw there was some confusion and some of what was said was incorrect. Some people thought our ICP is a company, while others thought it was a person, and there were some different ideas floating around.”

Reviewer 3: “When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “While there were still debates about some of the finer details and who some of the buyer personas may be, it was clear that our ICP was our guiding light.”

> “When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “Then I saw there was some confusion and some of what was said was incorrect. Some people thought our ICP is a company, while others thought it was a person, and there were some different ideas floating around.”

Author: ```suggestion
“When I joined Puzzle I was very happy to hear people talking about our persona,” says Scott. “Then I saw there was some confusion and some of what was said was incorrect. Some people thought our persona was a company, while others thought it was a person, and there were some different ideas floating around.”
```

> At PostHog, we’ve solved this issue with [a public handbook](/handbook) which documents [our ICP](/handbook/who-we-are-building-for) — but Puzzle took a different approach.  

Author: ```suggestion
At PostHog, we’ve solved this issue with [a public handbook](/handbook) which documents [who we are building for](/handbook/who-we-are-building-for) — but Puzzle took a different approach.  
```

> “When I joined Puzzle I was very happy to hear people talking about our persona,” says Scott. “Then I saw there was some confusion and some of what was said was incorrect. Some people thought our persona was a company, while others thought it was a person, and there were some different ideas floating around.”
> 

Author: ```suggestion
“When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “While there were still debates about some of the finer details and who some of the buyer personas may be, it was clear that our ICP was our guiding light.”

```

> “We have these all-hands meetings where everybody has to put their hand up and swear that our ICP is a founder and not an accountant,” says Scott. 

Reviewer 3: @joethreepwood would love if we could tweak this a bit. It was kind of an offhanded comment so maybe there's a better message we can use here?

> “We have these all-hands meetings where everybody has to put their hand up and swear that our ICP is a founder and not an accountant,” says Scott. 

Author: ```suggestion
“We have these all-hands meetings where everybody has to put their hand up and swear that our persona is a founder and not an accountant,” says Scott. 
```

> “When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “While there were still debates about some of the finer details and who some of the buyer personas may be, it was clear that our ICP was our guiding light.”

Reviewer 2: Maybe ICP vs persona needs to be briefly explained at the start.

Author: ```suggestion
“When I joined Puzzle I was very happy to hear people talking about our target persona,” says Scott. “While there were still debates about some of the finer details and who some of the buyer personas may be, it was clear that our target persona was our guiding light.”
```

> “We have these all-hands meetings where everybody has to put their hand up and swear that our persona is a founder and not an accountant,” says Scott. 

Author: ```suggestion
“We reinforce who we're building for constantly," says Scott. "We bring it up in all-hands regularly, for example, and will often reference it in other meetings. Internally, we're always trying to be clear that we're building primarily for founders." 
```

@scormier how's this?

Reviewer 3: Nailed it. Thanks @joethreepwood.

> **Putting it into practice:** There are lots of ways to disseminate information internally, from intranets to incorporating it into employee onboarding. Whichever route you take though, the key is consistency. 

## 5. Once you start, don’t stop

> Once you’ve established your ICP you must constantly maintain it. You need to always be speaking to users, working to understand their concerns, and analyzing the data behind their usage. In doing so you can find new segments to target, new needs to address, and new product improvements to make.

Author: ```suggestion
Once you’ve created your buyer persona you must constantly maintain it. You need to always be speaking to users, working to understand their concerns, and analyzing the data behind their usage. In doing so you can find new segments to target, new needs to address, and new product improvements to make.
```

> “We started looking at our biggest user drop-offs and found many ICP users were dropping off when they had to connect their bank,” says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”

Reviewer 3: “While we knew we had some drop-off within our onboarding funnel, we wanted to ensure that we weren’t bringing high-intent new customers into a ‘leaky bucket’. We needed to ensure that people who completed onboarding could quickly see the value of Puzzle and activate. After digging into our data in Posthog we could see that our activation rate wasn’t what we needed it to be”, says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”

> “We started looking at our biggest user drop-offs and found many ICP users were dropping off when they had to connect their bank,” says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”

Author: ```suggestion
“We started looking at our biggest user drop-offs and found many users that fit our persona were dropping off when they had to connect their bank,” says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”
```

> Once you’ve created your buyer persona you must constantly maintain it. You need to always be speaking to users, working to understand their concerns, and analyzing the data behind their usage. In doing so you can find new segments to target, new needs to address, and new product improvements to make.

Reviewer 2: ```suggestion
Once you’ve created your buyer persona you must constantly maintain it. You need to always be speaking to users, working to understand their concerns, and analyzing the data behind their usage. In doing so, you can find new segments to target, new needs to address, and new product improvements to make.
```

> “We started looking at our biggest user drop-offs and found many users that fit our persona were dropping off when they had to connect their bank,” says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”
> 

Author: ```suggestion
“While we knew we had some drop-off within our onboarding funnel, we wanted to ensure that we weren’t bringing high-intent new customers into a ‘leaky bucket’. We needed to ensure that people who completed onboarding could quickly see the value of Puzzle and activate. After digging into our data in Posthog we could see that our activation rate wasn’t what we needed it to be”, says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”

```

> Scott’s team learned that customers were becoming frustrated by delays during this connection step. As a result they improved the content on the page, added new loading animations, and surfacing data more quickly — an ICP-informed change which had an immediate impact on overall activation. 

Reviewer 3: Scott’s team learned that customers were becoming frustrated by delays before they could see their dashboard after completing onboarding. As a result they brought users into the app as it was loading so they could quickly see what they were waiting for. This reduced time to value dramatically and was a clear example of an ICP-informed change which had an immediate impact on overall activation.

> Scott’s team learned that customers were becoming frustrated by delays during this connection step. As a result they improved the content on the page, added new loading animations, and surfacing data more quickly — an ICP-informed change which had an immediate impact on overall activation. 

Author: ```suggestion
Scott’s team learned that customers were becoming frustrated by delays during this connection step. As a result they improved the content on the page, added new loading animations, and surfacing data more quickly — a persona-informed change which had an immediate impact on overall activation. 
```

> Scott’s team learned that customers were becoming frustrated by delays during this connection step. As a result they improved the content on the page, added new loading animations, and surfacing data more quickly — a persona-informed change which had an immediate impact on overall activation. 

Author: ```suggestion
Scott’s team learned that customers were becoming frustrated by delays before they could see their dashboard after completing onboarding. As a result they brought users into the app as it was loading so they could quickly see what they were waiting for. This reduced time to value dramatically and was a clear example of an ICP-informed change which had an immediate impact on overall activation.
```

> Scott’s team learned that customers were becoming frustrated by delays between completing onboarding and seeing their dashboard. As a result they brought users into the app as it was loading so they could quickly see what they were waiting for. This reduced time to value dramatically and was a clear example of an ICP-informed change which had an immediate impact on overall activation.

Author: ```suggestion
Scott’s team learned that customers were becoming frustrated by delays between completing onboarding and seeing their dashboard. As a result they brought users into the app as it was loading so they could quickly see what they were waiting for. This reduced time to value dramatically and was a clear example of an persona-informed change which had an immediate impact on overall activation.
```

> Scott’s team learned that customers were becoming frustrated by delays before they could see their dashboard after completing onboarding. As a result they brought users into the app as it was loading so they could quickly see what they were waiting for. This reduced time to value dramatically and was a clear example of an ICP-informed change which had an immediate impact on overall activation.

Reviewer 2: ```suggestion
Scott’s team learned that customers were becoming frustrated by delays between completing onboarding and seeing their dashboard. As a result they brought users into the app as it was loading so they could quickly see what they were waiting for. This reduced time to value dramatically and was a clear example of an ICP-informed change which had an immediate impact on overall activation.
```
